docs/user/metrics/boolean.md

```diff
@@ -144,6 +144,42 @@ assert!(flags::a11y_enabled.test_get_value(None).unwrap());
 
 </div>
 
+<div data-lang="C++" class="tab">
+
+> **Note**: C++ APIs are only available in Firefox Desktop.
+
+```c++
+#include "mozilla/glean/GleanMetrics.h"
+
+mozilla::glean::flags::a11y_enabled.Set(false);
+```
+
+There are test APIs available too:
+
+```c++
+#include "mozilla/glean/GleanMetrics.h"
+
+ASSERT_EQ(false, mozilla::glean::flags::a11y_enabled.TestGetValue().value());
+```
+
+</div>
+
+<div data-lang="JS" class="tab">
+
+> **Note**: JS APIs are only available in Firefox Desktop.
```

Comment:
I think we can already drop the note here for this. @brizental what do you think? The same API is already available with Glean.js and @chutten already wrote the docs, saving you the trouble :)

Reply:
> The same API is already available with Glean.js

The API is already implemented in Glean.js, but Glean.js itself is not available so I am scared of putting it out there that is "is available".

I do think this note should mention Glean.js somehow. Off the top of my head

```suggestion
> **Note**: JS APIs are currently only available in Firefox Desktop. Unpriviledged Javascript support is coming soon through the Glean.js project, follow the [Glean.js Roadmap](https://docs.google.com/document/d/1ClyXXWMxA1qf8OLDHPqzy3XOjb6JYVzwPablC_GNBYQ/edit?usp=sharing) for updates.
```

Reply:
Probably something for @badboy and @mdboom and @travis79 to have opinions on as well.

Reply:
Hm, maybe instead of the full roadmap just link to the repository? (and put a note in the repository's readme that it's still in development and not yet for public consumption?)

Reply:
100% agree, that is much better.